diff --git a/drivers/pci/pwrctrl/Kconfig b/drivers/pci/pwrctrl/Kconfig
index 1387cbdffcf8..7631aca134d1 100644
--- a/drivers/pci/pwrctrl/Kconfig
+++ b/drivers/pci/pwrctrl/Kconfig
@@ -8,6 +8,7 @@ config PCI_PWRCTRL
 
 config PCI_PWRCTRL_PWRSEQ
 	tristate
+	default y if ARCH_QCOM
 	select POWER_SEQUENCING
 	select PCI_PWRCTRL
 
diff --git a/drivers/soc/qcom/Kconfig b/drivers/soc/qcom/Kconfig
index 99cb655bd56a..029d3d6d63c4 100644
--- a/drivers/soc/qcom/Kconfig
+++ b/drivers/soc/qcom/Kconfig
@@ -126,6 +126,7 @@ config QCOM_AOSS_QMP
 
 config QCOM_QMI_HELPERS
 	tristate
+	default y if ARCH_QCOM
 	depends on NET
 
 config QCOM_RAMP_CTRL
